assets/prototype/domain/eventEditor/datetimes/dateList/DateList.js

```diff
@@ -61,6 +62,7 @@ const DateList = () => {
 	return (
 		<div>
 			{header}
+			<DatesListFilterBar datetimes={datetimes} />
```

Comment:
oh I just realized that `DatesListFilterBar` is not actually doing anything with the `datetimes` prop...
should it be? or is this prop here a carray over from EDTR v1 but can now be removed?

Reply:
Yes, this was a carry-over or I tried to experiment smth with lists, removed for now.